doc/guides/connection_management.rst

```diff
@@ -555,3 +555,51 @@ can then be given as arguments to the `SetStatus()` functions:
       'delay': 1.0,
       'source': 1,
       'receptor': 0}]
+
+
+Connecting sparse matrices with array indexing
```

Comment:
I think this section should maybe be placed further up, after connecting spatial networks maybe? To make the flow of the document more natural

Reply:
I've moved it.